etc/profile-m-z/tesseract.profile

```diff
@@ -0,0 +1,63 @@
+# Firejail profile for tesseract
+# Description: An OCR program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include tesseract.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+whitelist ${DOCUMENTS}
+whitelist ${DOWNLOADS}
+whitelist ${PICTURES}
```

Comment:
Documents and Pictures are blacklisted by disable-xdg.inc

Reply:
Nice catch! Fixed now.